portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,22 @@
  * Adds comments from /data
  */
 async function getComments() {
-  const commentsData = await fetch('/data');
+  max = document.getElementById('max').value
+  const commentsData = await fetch('/data?max=' + max);
   const comments = await commentsData.json();
   const ul = document.createElement('ul');
   for (const comment of comments) {
     const li = document.createElement('li');
     li.innerText = comment;
     ul.appendChild(li);
   }
+  document.getElementById("comments-container").innerHTML = "";
   document.getElementById('comments-container').appendChild(ul);
```

Comment:
single quotes https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
This will be resolved when I merge in the last branch. My git workflow should be easier to follow this week